sdk/src/types/block/signature/mod.rs

```diff
@@ -24,7 +26,13 @@ use crate::types::block::Error;
 pub enum Signature {
     /// An Ed25519 signature.
     #[packable(tag = Ed25519Signature::KIND)]
-    Ed25519(Ed25519Signature),
+    Ed25519(Box<Ed25519Signature>),
```

Comment:
Was clippy complaining about it or what was the issue exactly?

Reply:
Yes